prow/plugins/projectmanager/projectmanager.go

```diff
@@ -237,12 +275,15 @@ func getColumnID(gc githubClient, orgRepoName, projectName, columnName, issueURL
 	return nil, fmt.Errorf("could not find project %s in org/repo %s", projectName, orgRepoName)
 }
 
-func addPRToColumn(gc githubClient, columnID int, pe github.PullRequestEvent) error {
+func addPRToColumn(gc githubClient, columnID int, e eventData) error {
```

Comment:
nit: addIssueToColumn?

Reply:
Thanks for the review, please review to merge.

Reply:
/unhold